src/cyw43_ll.c

```diff
@@ -2014,6 +2014,8 @@ int cyw43_ll_wifi_join(cyw43_ll_t *self_in, size_t ssid_len, const uint8_t *ssid
         wpa_auth = CYW43_WPA2_AUTH_PSK;
     } else if (auth_type == CYW43_AUTH_WPA_TKIP_PSK) {
         wpa_auth = CYW43_WPA_AUTH_PSK;
+    } else if (auth_type == 0) {
+        wpa_auth = CYW43_AUTH_OPEN;
```

Comment:
I think the constants should be the other way around, ie:
```c
} else if (auth_type == CYW43_AUTH_OPEN) {
    wpa_auth = 0;
```
That will be consistent with the other if conditions here, checking against `CYW43_AUTH_xxx` constants.

Reply:
This seems good to me.